COLLABORATOR_GUIDE.md

```diff
@@ -452,7 +455,10 @@ Deprecations may land in a Node.js minor release but must not be upgraded to
 a Runtime Deprecation until the next major release.)
 
 No API can be moved to End-of-life without first having gone through a
-Runtime Deprecation cycle.
+Runtime Deprecation cycle. However, there is no requirement that deprecated
+code must progress ultimately to *End-of-Life*. It is possible, and at times
+preferable, for code that is documentation-only or runtime deprecated to
+remain so indefinitely.
```

Comment:
`It is possible, and at times preferable, for code that is documentation-only or runtime deprecated to remain so indefinitely.` -> `Documentation-only and runtime deprecations may remain indefinitely.`